pygdbmi/tests/response_samples.txt

```diff
@@ -0,0 +1,56 @@
+{'stream': 'stdout', 'type': 'notify', 'message': 'breakpoint-modified', 'token': None, 'payload': {'bkpt': {'line': '81', 'fullname': '/build/eglibc-MjiXCM/eglibc-2.19/io/../sysdeps/unix/syscall-template.S', 'file': '../sysdeps/unix/syscall-template.S', 'times': '10', 'original-location': 'write', 'number': '2', 'addr': '0x00007fb564e65f70', 'type': 'breakpoint', 'enabled': 'y', 'disp': 'keep', 'thread-groups': ['i1']}}}
```

Comment:
I think ensuring the buffering works in a randomized way is a good approach, since the controller needs to be able to deal with output that's randomly truncated, but I do not understand why this file is structured as dictionaries and not a large string of gdb output that is read from 1 to n bytes at a time until the entire string is consumed.